src/init.cpp

```diff
@@ -1250,6 +1253,26 @@ bool AppInitLockDataDirectory()
     return true;
 }
 
+static void FlushAfterSync() 
+{
+    // Once initial sync is finished, flush state to protect against data loss
+    if (IsInitialBlockDownload()) {
+        scheduler.scheduleFromNow(*FlushAfterSync, SYNC_CHECK_INTERVAL * 1000);
```

Comment:
I don't understand the purpose of `*` before `FlushAfterSync`.

Reply:
Removed them.